### YamlMime:ManagedReference
items:
- uid: System.IdentityModel.RequestException
  id: RequestException
  children:
  - System.IdentityModel.RequestException.#ctor
  - System.IdentityModel.RequestException.#ctor(System.String)
  - System.IdentityModel.RequestException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  - System.IdentityModel.RequestException.#ctor(System.String,System.Exception)
  langs:
  - csharp
  name: RequestException
  nameWithType: RequestException
  fullName: System.IdentityModel.RequestException
  type: Class
  summary: "Klasa podstawowa dla wyjątków zgłoszonych w błędów żądań."
  syntax:
    content: 'public abstract class RequestException : Exception'
  inheritance:
  - System.Exception
  implements: []
  inheritedMembers: []
  platform:
  - net462
- uid: System.IdentityModel.RequestException.#ctor
  id: '#ctor'
  parent: System.IdentityModel.RequestException
  langs:
  - csharp
  name: RequestException()
  nameWithType: RequestException.RequestException()
  fullName: System.IdentityModel.RequestException.RequestException()
  type: Constructor
  assemblies:
  - System.IdentityModel
  namespace: System.IdentityModel
  summary: "Inicjuje nowe wystąpienie klasy <xref href=&quot;System.IdentityModel.RequestException&quot;> </xref> klasy."
  remarks: "Ten konstruktor inicjuje <xref:SystemException.Message>Właściwość nowego wystąpienia systemu dostarczanego przez komunikatu opisującego błąd.</xref:SystemException.Message> Ten komunikat bierze pod uwagę bieżące ustawienia kulturowe systemu.       W poniższej tabeli przedstawiono początkowe wartości właściwości dla nowego wystąpienia.      | Właściwość | Wartość |   |--------------|-----------|   | <xref:System.Exception.InnerException>| `null`.|   | <xref:System.Exception.Message>| Ciąg z komunikatem o zlokalizowane. |</xref:System.Exception.Message> </xref:System.Exception.InnerException>"
  syntax:
    content: protected RequestException ();
    parameters: []
  overload: System.IdentityModel.RequestException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.IdentityModel.RequestException.#ctor(System.String)
  id: '#ctor(System.String)'
  parent: System.IdentityModel.RequestException
  langs:
  - csharp
  name: RequestException(String)
  nameWithType: RequestException.RequestException(String)
  fullName: System.IdentityModel.RequestException.RequestException(String)
  type: Constructor
  assemblies:
  - System.IdentityModel
  namespace: System.IdentityModel
  summary: "Inicjuje nowe wystąpienie klasy <xref href=&quot;System.IdentityModel.RequestException&quot;> </xref> klasy z powodu określonego błędu."
  remarks: "Ten konstruktor inicjuje <xref:System.Exception.Message>Właściwość nowego przy użyciu wyjątek `message` parametru.</xref:System.Exception.Message> Zawartość `message` ma na celu rozumiane przez człowieka. Element wywołujący tego konstruktora jest wymagana, aby upewnić się, że ten ciąg został zlokalizowany dla bieżącego ustawienia kulturowego systemu.       W poniższej tabeli przedstawiono początkowe wartości właściwości dla nowego wystąpienia.      | Właściwość | Wartość |   |--------------|-----------|   | <xref:System.Exception.InnerException>| `null`.|   | <xref:System.Exception.Message>| Ciąg z komunikatem o. |</xref:System.Exception.Message> </xref:System.Exception.InnerException>"
  syntax:
    content: protected RequestException (string message);
    parameters:
    - id: message
      type: System.String
      description: "Komunikat o błędzie opisujący przyczynę wyjątek."
  overload: System.IdentityModel.RequestException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.IdentityModel.RequestException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  id: '#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)'
  parent: System.IdentityModel.RequestException
  langs:
  - csharp
  name: RequestException(SerializationInfo,StreamingContext)
  nameWithType: RequestException.RequestException(SerializationInfo,StreamingContext)
  fullName: System.IdentityModel.RequestException.RequestException(SerializationInfo,StreamingContext)
  type: Constructor
  assemblies:
  - System.IdentityModel
  namespace: System.IdentityModel
  summary: "Inicjuje nowe wystąpienie klasy <xref href=&quot;System.IdentityModel.RequestException&quot;> </xref> z zserializowanymi danymi."
  remarks: "Ten konstruktor jest wywoływana podczas deserializacji do odtworzenia obiekt wyjątku przesyłane przez strumień."
  syntax:
    content: protected RequestException (System.Runtime.Serialization.SerializationInfo info, System.Runtime.Serialization.StreamingContext context);
    parameters:
    - id: info
      type: System.Runtime.Serialization.SerializationInfo
      description: "A <xref href=&quot;System.Runtime.Serialization.SerializationInfo&quot;> </xref> obiekt przechowujący dane serializacji obiektu."
    - id: context
      type: System.Runtime.Serialization.StreamingContext
      description: "A <xref:System.Runtime.Serialization.StreamingContext>obiekt zawierający informacje kontekstowe dotyczące źródła lub miejsca docelowego.</xref:System.Runtime.Serialization.StreamingContext>"
  overload: System.IdentityModel.RequestException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.IdentityModel.RequestException.#ctor(System.String,System.Exception)
  id: '#ctor(System.String,System.Exception)'
  parent: System.IdentityModel.RequestException
  langs:
  - csharp
  name: RequestException(String,Exception)
  nameWithType: RequestException.RequestException(String,Exception)
  fullName: System.IdentityModel.RequestException.RequestException(String,Exception)
  type: Constructor
  assemblies:
  - System.IdentityModel
  namespace: System.IdentityModel
  summary: "Inicjuje nowe wystąpienie klasy <xref href=&quot;System.IdentityModel.RequestException&quot;> </xref> z określonego komunikatu o błędzie i odwołania do wyjątek wewnętrzny będący przyczyną tego wyjątku."
  remarks: "Wyjątek zgłaszany bezpośrednio w wyniku poprzedniego wyjątku powinien zawierać odwołanie do poprzedniego wyjątku we <xref:System.Exception.InnerException>Właściwości.</xref:System.Exception.InnerException> <xref:System.Exception.InnerException>Właściwość zwraca tę samą wartość, która została przekazana do konstruktora.</xref:System.Exception.InnerException>       W poniższej tabeli przedstawiono początkowe wartości właściwości dla nowego wystąpienia.      | Właściwość | Wartość |   |--------------|-----------|   | <xref:System.Exception.InnerException>| Wyjątek wewnętrzny odwołania. |   | <xref:System.Exception.Message>| Ciąg z komunikatem o. |</xref:System.Exception.Message> </xref:System.Exception.InnerException>"
  syntax:
    content: protected RequestException (string message, Exception innerException);
    parameters:
    - id: message
      type: System.String
      description: "Komunikat o błędzie opisujący przyczynę wyjątek."
    - id: innerException
      type: System.Exception
      description: "<xref:System.Exception>Będący przyczyną bieżącego wyjątku.</xref:System.Exception> Jeśli `innerException` parametr nie jest <xref uid=&quot;langword_csharp_null&quot; name=&quot;null&quot; href=&quot;&quot;> </xref>, bieżący wyjątek jest zgłaszany w <xref uid=&quot;langword_csharp_catch&quot; name=&quot;catch&quot; href=&quot;&quot;> </xref> bloku, który obsługuje wyjątek wewnętrzny."
  overload: System.IdentityModel.RequestException.#ctor*
  exceptions: []
  platform:
  - net462
references:
- uid: System.Exception
  isExternal: false
  name: System.Exception
- uid: System.IdentityModel.RequestException.#ctor
  parent: System.IdentityModel.RequestException
  isExternal: false
  name: RequestException()
  nameWithType: RequestException.RequestException()
  fullName: System.IdentityModel.RequestException.RequestException()
- uid: System.IdentityModel.RequestException.#ctor(System.String)
  parent: System.IdentityModel.RequestException
  isExternal: false
  name: RequestException(String)
  nameWithType: RequestException.RequestException(String)
  fullName: System.IdentityModel.RequestException.RequestException(String)
- uid: System.String
  parent: System
  isExternal: true
  name: String
  nameWithType: String
  fullName: System.String
- uid: System.IdentityModel.RequestException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  parent: System.IdentityModel.RequestException
  isExternal: false
  name: RequestException(SerializationInfo,StreamingContext)
  nameWithType: RequestException.RequestException(SerializationInfo,StreamingContext)
  fullName: System.IdentityModel.RequestException.RequestException(SerializationInfo,StreamingContext)
- uid: System.Runtime.Serialization.SerializationInfo
  parent: System.Runtime.Serialization
  isExternal: false
  name: SerializationInfo
  nameWithType: SerializationInfo
  fullName: System.Runtime.Serialization.SerializationInfo
- uid: System.Runtime.Serialization.StreamingContext
  parent: System.Runtime.Serialization
  isExternal: true
  name: StreamingContext
  nameWithType: StreamingContext
  fullName: System.Runtime.Serialization.StreamingContext
- uid: System.IdentityModel.RequestException.#ctor(System.String,System.Exception)
  parent: System.IdentityModel.RequestException
  isExternal: false
  name: RequestException(String,Exception)
  nameWithType: RequestException.RequestException(String,Exception)
  fullName: System.IdentityModel.RequestException.RequestException(String,Exception)
- uid: System.IdentityModel.RequestException.#ctor*
  parent: System.IdentityModel.RequestException
  isExternal: false
  name: RequestException
  nameWithType: RequestException.RequestException
